Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -94,11 +95,16 @@ static const ReportFeatures reporting_mode_features[] = {
     {2, 0, 10, 9, 23},
     // 0x37: Core Buttons and Accelerometer with 10 IR bytes and 6 Extension Bytes
     {2, 3, 10, 6, 23},
-    // UNSUPPORTED (but should be easy enough to implement):
+    // Ugly padding members so 0x3d,3e,3f are properly placed in the array:
+    {0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0},
+    {0, 0, 0, 0, 0},
     // 0x3d: 21 Extension Bytes
     {0, 0, 0, 21, 23},
     // 0x3e / 0x3f: Interleaved Core Buttons and Accelerometer with 36 IR bytes
-    {0, 0, 0, 0, 23},
+    {2, 1, 0, 18, 23},
```

Comment:
Since you added padding before this, and that last entry represents two reporting modes...should this line be duplicated?
The access in `Wiimote::Update` might go out of bounds otherwise (unless I missed some context not shown in the web diff on GitHub)

Reply:
Uh yeah.. Good call. I plan on reworking that whole array in general to make it less ugly or at least contain the ugliness a bit more.